### ISSUE TITLE
Formatting on EOF tokens produces a bad syntax output

### ISSUE THREAD

**Issue by dlsniper** [VladRassokhin/intellij-hcl#8]
Hi,

Given the following example:
```hcl
resource "aws_iam_role_policy" "test_policy" {
    name = "test_policy"
    role = "${aws_iam_role.test_role.id}"
    policy = <<EOF
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Action": [
        "ec2:Describe*"
      ],
      "Effect": "Allow",
      "Resource": "*"
    }
  ]
}
EOF
}

resource "aws_iam_role" "test_role" {
    name = "test_role"
    assume_role_policy = <<EOF
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Action": "sts:AssumeRole",
      "Principal": {
        "Service": "ec2.amazonaws.com"
      },
      "Effect": "Allow",
      "Sid": ""
    }
  ]
}
EOF
}
```

the EOF token is not registered correctly and thus the formatter is wrong (and highlighter as well).

It would be cool if `<<EOF - EOF` would be marked as a string so and injections would be provided so that JSON could be inserted as a language there thus providing highlighting there as well.

Thank you for the work on the plugin!

**Comment by VladRassokhin** [comment]
Thank you for report, that's something new, never heard of heredoc in terraform, will investigate.

**Comment by dlsniper** [comment]
The example is taken from : https://www.terraform.io/docs/providers/aws/r/iam_role_policy.html if it helps. Thank you!

**Comment by VladRassokhin** [comment]
Ok, I've missed one of long existing language features, https://github.com/hashicorp/hcl/issues/6 
But there no sign of heredocs in '.y' files in hcl source code, so need to read all parsing code.

Will try to fix that in next release.
Thank for example, 'll add as test.

**Comment by VladRassokhin** [comment]
Fixed, available in 0.2.6, it's now available in plugins repo

**Comment by dlsniper** [comment]
Awesome, thank you!
